boards/remote/Makefile.include

```diff
@@ -0,0 +1,39 @@
+# define the cpu used by the Re-mote board
+export CPU       =  cc2538
+export CPU_MODEL ?= cc2538sf53
+
+# define the default port depending on the host OS
+PORT_LINUX  ?= /dev/ttyUSB1
+PORT_DARWIN ?= $(shell ls -1 /dev/tty.usbserial-* | head -n 1)
+
```

Comment:
You could use the `wildcard` makefile function for the same purpose, except
it does not depend on the output formatting of any shell commands.